guide/src/commands/path-setup/index.md

```diff
@@ -21,10 +21,12 @@ DESCRIPTION:
     Create objects (client, connection, or channel) on chains
 
 SUBCOMMANDS:
-    help       Get usage information
-    client     Create a new IBC client
-    connection Create a new connection between two chains
-    channel    Create a new channel between two chains
+    channel       Create a new channel between two chains using a pre-existing connection.
+                      Alternatively, create a new client and a new connection underlying the new
+                      channel if a pre-existing connection is not provided
```

Comment:
This looks too long for the subcommand summary.

Can we put it under the command's long help, leaving the summary as one line?

Reply:
By having an `about` and `long_about` for the `create channel` subcommand will display only the short help (`about`). Meaning the commands `hemres create -h` and `hermes create --help` will both display:
```
Create objects (client, connection, or channel) on chains

USAGE:
    hermes create <SUBCOMMAND>

OPTIONS:
    -h, --help    Print help information

SUBCOMMANDS:
    channel       Create a new channel between two chains using a pre-existing connection
    client        Create a new IBC client
...
```
`--help` gives precedence to `about` over `long_about`, https://github.com/clap-rs/clap/issues/3193#issuecomment-996390269

But the command `hermes create channel -h` will display the short help:
```
Create a new channel between two chains using a pre-existing connection

USAGE:
    hermes create channel [OPTIONS] --a-chain <A_CHAIN_ID> --a-port <A_PORT_ID> --b-port <B_PORT_ID>
...
```
And the command `hermes create channel --help` will display the long help:
```
Create a new channel between two chains using a pre-existing connection.

Alternatively, create a new client and a new connection underlying the new channel if a pre-existing
connection is not provided.

USAGE:
    hermes create channel [OPTIONS] --a-chain <A_CHAIN_ID> --a-port <A_PORT_ID> --b-port <B_PORT_ID>
```
I think having the short version only for the `hermes create -h` and `hermes create --help`, and the short for `hermes create channel -h` and long for `hermes create channel --help` is a good option.
What do you think ?

Reply:
This is exactly how I think it should work, thanks.

Reply:
Dunno if help through doc comments automagically allows this, though, or whether it needs an explicit clap attribute.
What happens if the first line is set into its own paragraph?

Reply:
Having an empty line in the doc comments works. Like this:
```
    /// Create a new channel between two chains using a pre-existing connection.
    ///
    /// Alternatively, create a new client and a new connection underlying
    /// the new channel if a pre-existing connection is not provided.
    Channel(CreateChannelCommand),
```